recorder/src/main/resources/tracking.js

```diff
@@ -73,6 +73,7 @@ jsflight.TrackHash = function(event) {
         console.log(e);
     } finally {
         jsflight.eventId++;
+        jsflight.rebindAllHandlers();
```

Comment:
It is a pity, yeah, but IE8 has passed EndOfLife a long time ago. So, I think it's fair to set IE8 free of this feature, okay?